<launch>
  <!--<node name="rviz" pkg="rviz" type="rviz" args="-d $(find MSTC_Star_update)/rviz/navigate.rviz" />-->

  <param name = "/allocate_method" value = "MSTC" />
  <param name = "/MST_shape" value = "DINIC" />
  <param name = "/useROSPlanner" value = "true" />
  <param name = "/coverAndReturn" value = "false" />

  <param name = "/GA_max_iter" value = "200000" />
  <param name = "/unchanged_iter" value = "2000" />

  <param name = "/hp_max_iter" value = "5000" />

  <node name="coverage_path_server" pkg="TMSTC_Star" type="coverage_path_server" />
</launch>